controller/replica_controller.go

```diff
@@ -297,11 +297,18 @@ func (rc *ReplicaController) CreatePodSpec(obj interface{}) (*v1.Pod, error) {
 	if !ok {
 		return nil, fmt.Errorf("BUG: invalid object for engine pod spec creation: %v", r)
 	}
-	cmd := []string{
-		"launch", "replica",
+
+	cmd := []string{"launch", "replica"}
+	if r.Spec.BaseImage != "" {
+		cmd = []string{
+			"longhorn", "replica",
+			"--backing-file", "/base_image",
```

Comment:
Just use `{`longhorn`, `replica`}` for both cases should be fine. Nothing in the `launch` is for replica.